### PR TITLE
opera mini extreme mode style fix

### LINKED ISSUE #9818: Opera Mini Extreme style does not follow UX design.
**Describe the bug**
Opera Mini Extreme style does not follow UX design.

**To Reproduce**
Steps to reproduce the behaviour:

1. In Opera Mini activate the extreme mode
2. Go to: https://www.test.bbc.com/pidgin/articles/crrrkxz2k0ko
3. Go to: https://www.test.bbc.com/arabic/articles/cjn2e0jek55o 

**Expected behaviour**
Cards width should extend to the width of the page. 
There should be a spacing right equal to the spacing left at all breakpoints.

**Screenshots**
image

image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

- Device: Samsung Galaxy A5 2017
- OS: Android 8.0.0
- Browser Opera Mini Extreme mode
- Version 62.1.2254.60552

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by DarioR01**
Resolves #9818 

**Overall change:**
Style should follow UX designs (It is not possible to test this locally. Must be pushed to test or live for testing)

**Code changes:**

- Remove display flex. This is causing the box to not have the correct width.
- Removed scrollable logic in opera mini extreme. This was causing the page to scroll "outside" in rtl services. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by Louis-Matsika — APPROVED** [review]

File: src/app/components/ScrollablePromo/PromoList/index.jsx [hunk 1]
```diff
@@ -12,12 +12,13 @@ import useOperaMiniDetection from '#hooks/useOperaMiniDetection';
...
-  display: flex;
   list-style: none;
   ${({ dir }) => `padding-${dir === 'ltr' ? 'left' : 'right'}: 0;`}
   margin: 0;
-  ${({ isOperaMini }) => (isOperaMini ? `flex-direction: column;` : '')}
-
+  ${({ isOperaMini }) =>
+    isOperaMini
+      ? ``
+      : `display: flex;
```
Comment (MuhammadAHussain) [inline src/app/components/ScrollablePromo/PromoList/index.jsx]: Won't having `display:flex` cause problems in IE11 as it's only partially supported according to this https://caniuse.com/?search=display%3Aflex (we do still support IE11, right?)

Comment (DarioR01) [reply src/app/components/ScrollablePromo/PromoList/index.jsx]: We won't support IE11 anymore.
I added some code clarity changes. Nothing huge, the style is the same :D.

Comment (DarioR01) [reply src/app/components/ScrollablePromo/PromoList/index.jsx]: @aeroplanejane :
> IE11 is end of life in July - not a browser we’re actively supporting anymore

**Review by MuhammadAHussain — APPROVED** [review]
LGTM 👍

**Pull request merged**